sources/dylan/integer.dylan

```diff
@@ -241,23 +241,23 @@ define method \^
   end if
 end method \^;
 
-define function logior (#rest integers) => (logior :: <integer>)
+define sealed method logior (#rest integers) => (logior :: <integer>)
```

Comment:
Converting `logior` (and `logxor` and `logand`) to `sealed method` won't change anything, since they have no required arguments, so there is no dispatch happening.